custom_components/healthchecksio/__init__.py

```
@@ -94,3 +98,44 @@ async def async_unload_entry(
+    if version == 1:
+        v1to2: bool = await _migrate_1_to_2(hass, config_entry)
+        if not v1to2:
+            return False
+        version = 2
+
+    _LOGGER.info("Migration to version %s successful", version)
+    return True
+
+
+async def _migrate_1_to_2(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    CONF_CHECK = "check"
```

Pylint (3.3.7)

[convention] Variable name "CONF_CHECK" doesn't conform to snake_case naming style

(C0103)